src/crypto/tls/delegated_credentials_test.go

```diff
@@ -562,45 +546,39 @@ func testConnWithDC(t *testing.T, clientMsg, serverMsg string, clientConfig, ser
 	return false, nil
 }
 
+var count int
```

Comment:
This global counter makes me a bit nervous. Does it need to be global, or could it live in some function's local scope? If it must be global, could you add a comment documenting what it does, and which functions write to and read from it?

Reply:
It needs to be global to be an input to the callbacks for the certificate. The reason why it is is because the `getCertificate` callback does no accept any input other than the clienthelloinfo or criinfo. Let me see if I can change the design...

Reply:
I see. I would be fine with adding a comment documenting its purpose. Also, it would probably be useful for posterity if the name were more specific than "count".